docs/config/README.md

```diff
@@ -46,9 +46,11 @@ This is the list of prompt-wide configuration options.
 
 ### Options
 
-| Variable      | Default | Description                                    |
-| ------------- | ------- | ---------------------------------------------- |
-| `add_newline` | `true`  | Add a new line before the start of the prompt. |
+| Variable                | Default | Description                                                          |
+| ----------------------- | ------- | -------------------------------------------------------------------- |
+| `add_newline`           | `true`  | Add a new line before the start of the prompt.                       |
+| `prompt_order`          | `[]`    | Configure out custom prompt order modifying default order            |
+| `disable_default_order` | `false` | Disable default config order and only enable out custom prompt order |
```

Comment:
I don't think we really need the `disable_default_order` configuration option.
The default value of `prompt_order` could be retrieved from [PROMPT_ORDER](https://github.com/starship/starship/blob/master/src/print.rs#L10-L25), and overriding it in configuration would disable the default prompt order.

Please let me know if you have other functionality in mind around that configuration. 🙂

Reply:
I have nothing special functionality in mind. But what about adding some custom module `languages` which will add all supported languages instead of adding all languages in config file. So, starship can easily support new languages module which are added in future by default on new version of starship without modifying custom config file. User can also add one by one languages by themselves.

Reply:
If `language` custom module is good custom module for addition then it may be added in another PR separately rather then implementing in this PR.

Reply:
I think it would be a great idea, but getting the implementation right would be a challenge--you're essentially talking about implementing an extensions client for starship. We probably won't be ready to do that for some time.